build.assets/makefiles/base/network/network.mk

```diff
@@ -31,6 +31,6 @@ $(BINARIES): GOPATH := $(DIR)
 $(BINARIES):
 	mkdir -p $(DIR)/src/github.com/coreos
 	cd $(DIR)/src/github.com/coreos && git clone https://github.com/gravitational/flannel -b $(FLANNEL_VER) --depth 1
-	cd $(DIR)/src/github.com/coreos/flannel && go build -mod=vendor -o flanneld .
+	cd $(DIR)/src/github.com/coreos/flannel && GO111MODULE=on go build -mod=vendor -o flanneld .
```

Comment:
Why is this on in 5.5?

Reply:
Just porting https://github.com/gravitational/planet/pull/756 to fix failed gravity build.